source/src/processing/cs_FactoryReset.cpp

```diff
@@ -181,6 +184,12 @@ void FactoryReset::handleEvent(event_t& event) {
 		case CS_TYPE::EVT_MESH_FACTORY_RESET_DONE: {
 #if BUILD_MESHING == 1 && MESH_PERSISTENT_STORAGE == 1
```

Comment:
If we go for the `EVT_READY_FOR_FACTORY_RESET` with `FactoryResetClassBit` parameter, these ifdefs will disappear. :-)